src/main/java/org/spongepowered/api/scoreboard/Team.java

```diff
@@ -0,0 +1,190 @@
+     * Gets the name of this Team.
+     *
+     * @return Objective name
+     */
+    String getName();
+
+    /**
+     * Gets the name displayed to users for this team.
+     *
+     * @return Team display name
+     */
+    String getDisplayName();
+
+    /**
+     * Gets the color of this team.
+     *
+     * <p>The team's color is a distinct concept from its prefix or suffix.
+     * It is only used for colored sidebar display slots, and certain statistic
+     * criteria.</p>
+     *
+     * @return Team color
+     */
+    TextColor getColor();
+
+    /**
+     * Sets the color of this team.
+     *
+     * <p>The team's color is a distinct concept from its prefix or suffix.
+     * It is only used for colored sidebar display slots, and certain statistic
+     * criteria.</p>
+     *
+     * @param color The team color
+     * @throws IllegalArgumentException if color is {@link org.spongepowered.api.text.format.TextColors#RESET}
+     */
+    void setColor(TextColor color);
+
+    /**
+     * Sets the name displayed to users for this team.
+     *
+     * @param displayName the new display name
+     * @throws IllegalArgumentException if displayName is longer than 32
+     *     characters.
+     */
+    void setDisplayName(String displayName) throws IllegalArgumentException;
+
+    /**
+     * Gets the prefix prepended to the display name of users on this team.
+     *
+     * @return Team prefix
+     */
+    String getPrefix();
+
+    /**
+     * Sets the prefix prepended to the display name of users on this team.
+     *
+     * @param prefix New prefix
+     * @throws IllegalArgumentException if prefix is longer than 16
+     *     characters
+     */
+    void setPrefix(String prefix) throws IllegalArgumentException;
+
+    /**
+     * Gets the suffix appended to the display name of users on this team.
+     *
+     * @return the team's current suffix
+     */
+    String getSuffix();
+
+    /**
+     * Sets the suffix appended to the display name of users on this team.
+     *
+     * @param suffix the new suffix for this team.
+     * @throws IllegalArgumentException if suffix is longer than 16
+     *     characters
+     */
+    void setSuffix(String suffix) throws IllegalArgumentException;
+
+    /**
+     * Gets whether friendly fire is enabled.
+     *
+     * @return whether friendly fire is enabled
+     */
+    boolean allowFriendlyFire();
+
+    /**
+     * Sets whether friendly fire is enabled
+     *
+     * @param enabled true whether or not to enable friendly fire
+     */
+    void setAllowFriendlyFire(boolean enabled);
+
+    /**
+     * Gets whether invisible team members are shown.
+     *
+     * @return whether or not to show invisible team members 
+     */
+    boolean canSeeFriendlyInvisibles();
+
+    /**
+     * Sets whether invisible team members are shown.
+     *
+     * @param enabled whether or not to show invisible teammates
+     */
+    void setCanSeeFriendlyInvisibles(boolean enabled);
+
```

Comment:
While this may be intentional, you're missing the `nametagVisibility` and `deathMessageVisibility` options, which would both probably refer to a `TeamVisibility` enum. I think it would be an enum since it wouldn't make much sense to add new values, but that is debatable